packages/node/src/handlers.ts

```diff
@@ -62,11 +61,13 @@ export function tracingHandler(): (
       return next();
     }
 
-    // If there is a trace header set, we extract the data from it (parentSpanId, traceId, and sampling decision)
-    const traceparentData =
-      req.headers && isString(req.headers['sentry-trace']) && extractTraceparentData(req.headers['sentry-trace']);
-    const incomingBaggageHeaders = req.headers?.baggage;
-    const dynamicSamplingContext = baggageHeaderToDynamicSamplingContext(incomingBaggageHeaders);
+    const sentryTrace = req.headers && isString(req.headers['sentry-trace']) ? req.headers['sentry-trace'] : undefined;
```

Comment:
l: could be slightly simplified to:

```suggestion
    const sentryTrace = req.headers?.['sentry-trace'] || undefined;
```

?

Reply:
We have the `isString` check here because req.headers['sentry-trace'] can provide an array, and we currently don't consider that valid.

I'd rather come back and re-evaluate that decision for every header usage, but for now don't want to unblock this change in particular. Adding as a TODO to the GH issue.

Reply:
ah, I see, makes sense. 👍